File: data manipulation/product of all digit.py
takeNum = int(input("Enter an four-digit number: "))
lastNum = takeNum % 10
middleNum = (takeNum // 10) % 10
middleNum2 = (takeNum // 100) % 10
firstNum = takeNum // 1000
productDigits = firstNum * middleNum2 * middleNum * lastNum
print("The product of all digits is:", productDigits)

File: digitManipulation/productOfAllDigit.cpp
#include<iostream>
using namespace std;
int main()
{
    int number, firstDigit, secondDigit, thirdDigit, product;

    cout<<"Enter a three digit number :";
    cin>>number;

    if(number >= 100 && number <= 999)
    {
        firstDigit = number / 100;
        secondDigit = (number / 10) % 10;
        thirdDigit = number % 10;

        product = firstDigit * secondDigit * thirdDigit;

        cout<<"Product of the digits = "<<product;
        
    }
    else
    {
        cout<<"Please enter a three digit number.";
    }
    
    return 0;
}